src/main/scala/cromwell/engine/backend/runtimeattributes/CromwellRuntimeAttributes.scala

```diff
@@ -24,163 +29,187 @@ case class CromwellRuntimeAttributes(docker: Option[String],
 object CromwellRuntimeAttributes {
   private val log = LoggerFactory.getLogger("RuntimeAttributes")
 
-  def apply(wdlRuntimeAttributes: RuntimeAttributes, backendType: BackendType): CromwellRuntimeAttributes =
-    fromAttributeMap(AttributeMap(wdlRuntimeAttributes.attrs), backendType)
+  def apply(wdlRuntimeAttributes: RuntimeAttributes, backendCall: BackendCall, workflowOptions: Option[WorkflowOptions]): CromwellRuntimeAttributes = {
```

Comment:
I definitely agree, though I'm not sure what the best path forward for that is.  I know @mcovarr and @gauravs90 are currently working on an actor to do this kind of backend-specific validation.  I didn't want to try to reinvent too much considering that's coming soon